src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,18 @@
+.TodoInfo {
+  width: max-content;
```

Comment:
Consider using more descriptive class name instead of `.TodoInfo`. For example, `.todo-info-container` or `.todo-info-wrapper`. 🤔